### PR TITLE
LUT-25131

### PR DESCRIPTION
**Pull request by Timothee-Hx**
https://dev.lutece.paris.fr/bugtracker/issues/25131

Wikicreole to markdown

fix : when create wiki (parentpagename not null)

### CODE REVIEW

**Comment by Timothee-Hx** [comment]
Info for testing : 
Some features rely on Bootstrap, when testing make sure to use a recent version of lutece-core (to have a recent version of bootstrap). 

If the switch between branches of this plugin, make sure to disable and enable (or enable or disable) cache in you browser(right click; inspect, network, checkbox "disable cache", so your browser doesn't keep memory of elements from oder versions.

File: src/java/fr/paris/lutece/plugins/wiki/business/IImageDAO.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/business/IImageDAO.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/business/ITopicDAO.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/business/ITopicDAO.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/business/Image.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/business/Image.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/business/ImageDAO.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/business/ImageDAO.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/business/ImageHome.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/business/ImageHome.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/business/Topic.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/business/Topic.java:1]: Line added changes and one useless import.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/business/TopicVersion.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/business/TopicVersion.java:1]: Lines added changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/business/TopicVersionHome.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/business/TopicVersionHome.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/search/WikiSearchEngine.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/search/WikiSearchEngine.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/DiffService.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/DiffService.java:1]: Line deleted change.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/ImageService.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/ImageService.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/RoleService.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/RoleService.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/WikiLocaleService.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/WikiLocaleService.java:1]: Line added change and one useless import.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/WikiPlugin.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/WikiPlugin.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/WikiSitePropertiesGroup.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/WikiSitePropertiesGroup.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/WikiUtils.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/WikiUtils.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/parser/ParserOptions.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/parser/ParserOptions.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/parser/TextFormatMacro.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/parser/TextFormatMacro.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/parser/TextWrapperMacro.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/parser/TextWrapperMacro.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/parser/WikiMacro.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/parser/WikiMacro.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/service/parser/WikiMacroService.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/service/parser/WikiMacroService.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/java/fr/paris/lutece/plugins/wiki/utils/auth/WikiAnonymousAuthentication.java
Comment (ThomasDumont) [inline src/java/fr/paris/lutece/plugins/wiki/utils/auth/WikiAnonymousAuthentication.java:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/site/fr/xdoc/index.xml
Comment (ThomasDumont) [inline src/site/fr/xdoc/index.xml:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/site/site_fr.xml
Comment (ThomasDumont) [inline src/site/site_fr.xml:1]: Whitespace-only changes.
Remove from Pull Request.

File: src/site/xdoc/index.xml
Comment (ThomasDumont) [inline src/site/xdoc/index.xml:1]: Whitespace-only changes.
Remove from Pull Request.

File: webapp/WEB-INF/conf/plugins/wiki.properties
Comment (ThomasDumont) [inline webapp/WEB-INF/conf/plugins/wiki.properties:1]: Whitespace-only changes.
Remove from Pull Request.

File: webapp/WEB-INF/conf/plugins/wiki_context.xml
Comment (ThomasDumont) [inline webapp/WEB-INF/conf/plugins/wiki_context.xml:1]: Whitespace-only changes.
Remove from Pull Request.

File: webapp/WEB-INF/templates/skin/plugins/wiki/list_wiki.html
Comment (ThomasDumont) [inline webapp/WEB-INF/templates/skin/plugins/wiki/list_wiki.html:1]: Whitespace-only changes.
Remove from Pull Request.

File: webapp/WEB-INF/templates/skin/plugins/wiki/map_wiki.html
Comment (ThomasDumont) [inline webapp/WEB-INF/templates/skin/plugins/wiki/map_wiki.html:1]: Whitespace-only changes.
Remove from Pull Request.

File: webapp/WEB-INF/templates/skin/plugins/wiki/search_wiki.html
Comment (ThomasDumont) [inline webapp/WEB-INF/templates/skin/plugins/wiki/search_wiki.html:1]: Whitespace-only changes.
Remove from Pull Request.

File: webapp/js/plugins/wiki/prettify/run_prettify.js
Comment (ThomasDumont) [inline webapp/js/plugins/wiki/prettify/run_prettify.js:1]: Whitespace-only changes.
Remove from Pull Request.

**Review by ThomasDumont — COMMENTED** [review]
Several files that must be removed from this Pull Request.

**Comment by Timothee-Hx** [comment]
fixes made (sending you a email)

**Comment by Timothee-Hx** [comment]
re-open

**Pull request merged**